class.json-api-endpoints.php

```diff
@@ -1139,7 +1141,8 @@ function get_author( $author, $show_email = false ) {
 			$avatar_URL = $this->api->get_avatar_url( $email );
 		}
 
-		$email = $show_email ? (string) $email : false;
+		$email = $show_email_and_ip ? (string) $email : false;
+		$ip_address = $show_email_and_ip ? (string) $ip_address : false;
```

Comment:
Ok yeah, I guess it makes sense not to fix it if it's not broken :D 
The `if` could still be introduced though.